egs/wsj/s5/steps/nnet3/chain/get_egs.sh

```diff
@@ -65,6 +65,7 @@ online_ivector_dir=  # can be used if we are including speaker information as iV
 cmvn_opts=  # can be used for specifying CMVN options, if feature type is not lda (if lda,
             # it doesn't make sense to use different options than were used as input to the
             # LDA transform).  This is used to turn off CMVN in the online-nnet experiments.
+online_cmvn_frontend=false # set to 'true' to activate online cmvn,
```

Comment:
clarify: "(for the regular features; separate from the online CMVN used for the ivectors)"
and please remove the _frontend; I think online_cmvn is sufficient.